### ISSUE TITLE
Ordering of files

### ISSUE THREAD

**Issue by vikyaths** [cgauthier/karma_jasmine_2_extjs4#2]
Looks like the files need to be included in the order of usage in the code, i.e say I have a class Car which requires a class Steering, this class needs to be included in the config file of Karma (in the 'files' array) before adding the Car class file. 
I would like to know if this is indeed true or if I am missing something or if there is an alternative way of loading the dependent files just like how ExtJS does it at runtime.

Adding these files manually will be a tedious task if you were to write test cases for an existing application which has a huge no. of files. 

Welcome your views on this. Thank you.

**Comment by cgauthier** [comment]
Yes the order of files is important due to how JavaScript works.
